frontend/src/features/permits/pages/Application/ApplicationForm.tsx

```diff
@@ -272,7 +267,7 @@ export const ApplicationForm = ({
           trailerSubTypes={trailerSubTypes}
           companyInfo={companyInfo}
           durationOptions={PERMIT_DURATION_OPTIONS}
-          doingBusinessAs={doingBusinessAs}
+          doingBusinessAs={companyLegalName}
```

Comment:
I'm confused by this change. `companyLegalName` isn't always the same as `doingBusinessAs`, right?

Reply:
Melanie said that the DBA should always be displayed (regardless of whether or not staff is acting as company), so the doingBusinessAs would be equivalent to companyLegalName (in removed line 58)

Reply:
```
companyLegalName: "Parisian Trucking Corporation",
doingBusinessAs: "Paris Corp"
```

In this case where dba does exist, you're still using companyLegalName? If so, can you get it added to the specs (if not already)?

Reply:
I always thought companyLegalName was the DBA (except before - only show it when staff acting as company)... Btw where did you get the doingBusinessAs: "Paris Corp"? Also, how do I add to the specs?

Reply:
No, DBA is a different field altogether. We added it to the create company forms a few months back. I think you should tweak the value for DBA to something like 
`doingBusinessAs ?? companyLegalName`.

As far as specs are concerned, ping Michael to get it updated there.

Screenshot from staff create new company page, for example
<img width="370" alt="image" src="https://github.com/bcgov/onroutebc/assets/84348052/957fdda5-4734-4aeb-9995-c451d07a824e">

Reply:
In any case, I'd recommend reconfirming the requirements here.

Reply:
Currently having a chat with Michael, I stand corrected... It should indeed be the DBA field instead of companyLegalName (which was a wrong assumption on my part). I'll fix this.

Reply:
No need to change the specs as the spec is correct